raml/api.raml

```diff
@@ -111,6 +111,100 @@ mediaType: application/json
               schema: !include schemas/404.json
               example: !include response_examples/404/region.json
 
+    /runbooks:
```

Comment:
Could you please add response examples? At least for GET api calls with 200 response status.